src/views/Page404/components/ScottieDog/scottieDog.scss

```diff
@@ -0,0 +1,114 @@
+$dog-color: #555;
+$dog-body-width: 10em;
+$dog-body-height: 5em;
+
+.dog {
+    position: relative;
+    /*** font size of root element controls size of dog ***/
+    // font-size: 30px;
+
+    /*** Used for walking across screen animation (optional) ***/
+    animation: dog-walk linear 30s;
+    animation-delay: 5s;
+    left: 150%;
```

Comment:
It appears that this starts the dog off to the right of the screen.  That makes it take about 10 seconds, instead of 5 (animation-delay), for the dog to appear.  Is that intentional?  (Mentioning that in comment above would save people wondering.)

Dog is cute!